packages/graphiql/index.d.ts

```diff
@@ -0,0 +1,42 @@
+declare module 'graphiql' {
+  import React from 'react'
+  import { GraphQLSchema} from 'graphql'
```

Comment:
whats wrong with the formatting?

Reply:
A very subtle spacing error that you should not have to think about because prettier should do it for you:

```diff
-  import { GraphQLSchema} from 'graphql'
+  import { GraphQLSchema } from 'graphql'
```